web/src/components/series.astro

```diff
@@ -0,0 +1,462 @@
+          tags: ["int.", "tools"],
+          description:
+            "Operate NVIDIA FLARE with the commands and functionalities of the admin client, which allow you to manage your system.",
+          link: "https://nvflare.readthedocs.io/en/main/real_world_fl/operation.html",
+        },
+      ]
+    },
+    {
+      id: "103",
+      title: "103: Deployment",
+      description: "Provision and deploy a FLARE system.",
+      cards: [
+        {
+          title: "NVIDIA FLARE Dashboard",
+          tags: ["adv.", "deployment"],
+          description:
+            "Provides a CLI or UI to collect information of clients and users from different organizations, and generates startup kits with keys and certificates for users to download.",
+          link: "https://nvflare.readthedocs.io/en/main/user_guide/dashboard_api.html",
+        },
+        {
+          title: "Containerized Deployment with Docker",
+          tags: ["beg.", "deployment"],
+          description:
+            "Run NVIDIA FLARE in a Docker container.",
+          link: "https://nvflare.readthedocs.io/en/main/quickstart.html#containerized-deployment-with-docker",
+        },
+        {
+          title: "Cloud Deployment",
+          tags: ["adv.", "deployment"],
+          description:
+            "Deploy the NVIDIA FLARE dashboard, server, and clients in the cloud. Learn to use the provided scripts for AWS and Azure.",
+          link: "https://nvflare.readthedocs.io/en/main/real_world_fl/cloud_deployment.html",
+        },
+        {
+          title: "Deployment to Kubernetes",
+          tags: ["adv.", "deployment"],
+          description:
+            "Run NVIDIA FLARE inside Amazon Elastic Kubernetes Service (EKS).",
+          link: "https://nvflare.readthedocs.io/en/main/real_world_fl/kubernetes.html",
+        },
+        {
+          title: "Helm Chart Deployment",
+          tags: ["adv.", "deployment"],
+          description:
+            "Use the provisioning tool's HelmChartBuilder to generate a reference Helm Chart to deploy NVIDIA FLARE to a local microk8s Kubernetes instance.",
+          link: "https://nvflare.readthedocs.io/en/main/user_guide/helm_chart.html",
+        },
+        {
+          title: "Secure Provisioning",
+          tags: ["adv.", "deployment"],
+          description:
+            "Utilize the secure provisioning tool for streamlining the process to provision, start, and operate federated learning with a trusted setup.",
+          link: "https://nvflare.readthedocs.io/en/main/real_world_fl/overview.html",
+        },
+      ]
+    },
+  ]
+}
+
+const series_200 = {
+  id: "200",
+  title: "FLARE 200: Algorithms",
+  subseries_list: [
+    {
+      id: "201",
+      title: "201: FL Algorithms",
+      description: "See how FL algorithms can be implemented in a variety of frameworks.",
+      cards: [
+        {
+          title: "Step-by-Step CIFAR10 Examples",
+          tags: ["beg.", "algorithm", "pytorch", "dl"],
```

Comment:
what things are categorized as tag "algorithm"? we could also say all the examples are using different types of algorithm

Reply:
"algorithm" was from the tags in the tutorial catalog page, where we divided all examples in "algorithm", "tool", 
"deployment"

Reply:
got it, the series.astro now just want to be consistent with tutorial catalog page.
makes sense, we can thinking of any tagging enhancements in other PRs.